taskcluster/ci/post-beetmover/kind.yml

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: ac_taskgraph.loader.all_dep:loader
+
+transforms:
+    - ac_taskgraph.transforms.all_dep:transforms
+    - ac_taskgraph.transforms.post_dummy:transforms
+    - ac_taskgraph.transforms.chunk:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - beetmover
+
+group-by: build-type
```

Comment:
Likely a bug: I believe we should include: 

https://github.com/mozilla-mobile/android-components/blob/44987b3668e20a146b20749a8584e88d572fcfc5/taskcluster/ci/beetmover/kind.yml#L20-L22

just like we have on beetmover. I believe that explains why some beetmover tasks are pulled in the pull-request and the push-bors graph.

Reply:
Hm, this seems to have disappeared before this change. Maybe `all_dep` fixed it?

Reply:
Ah maybe!